static/js/message_events.js

```diff
@@ -306,6 +306,10 @@ exports.update_messages = function update_messages(events) {
         notifications.received_messages([msg]);
         alert_words.process_message(msg);
 
+        if ($('#message-edit-history').hasClass('in')) {
+            message_edit.get_and_render_message_history(msg);
+        }
```

Comment:
This needs to check that the message that was just edited is the same as the one we're currently looking at message-edit history for.  Otherwise, it'll generate wrong behavior.

My guess is we can also use something in `overlay.js` to check if the overlay is open?  But maybe that doesn't matter because we want to check if it's open to a given message, and doing that will require exporting some state from `message_edit.js`.

@sahil839 I'd like to get you into the habit of thinking about and testing corner cases like this -- it's really important to writing correct code.

Reply:
@timabbott I will definitely try to cover most of the corner cases in first time itself.

I was not able to find any exisiting way to check that edit history modal is of which message, so I added code to add `data-message-id` attr while openign the modal.